Formula/fastlane.rb

```diff
@@ -0,0 +1,55 @@
+class Fastlane < Formula
+  desc "Easiest way to build and release mobile apps"
+  homepage "https://fastlane.tools"
+  url "https://github.com/fastlane/fastlane/archive/2.137.0.tar.gz"
+  sha256 "ff3813d75bbced1030b5d7e3e714e75f358788f74d951be796515e8ec4e7ec79"
+  head "https://github.com/fastlane/fastlane.git"
+
+  RUBY_DEP = "ruby@2.5".freeze
+
+  depends_on RUBY_DEP
+
+  def install
+    # Build and unpack gem into lib directory to reference from our Gemfile
+    system "gem", "build", "fastlane.gemspec"
+    gem_name = Dir["fastlane-*.gem"].last
+    version_name = File.basename(gem_name, ".gem")
```

Comment:
```suggestion
    version_name = gem_name.basename
```

although it would be nice to use `#{version}` if possible instead.